#1274 Make share larger

harriseldon opened the pull request:
I added a zoom button to the network share with a new zoom dialog. The dialog shows the url, but maximizes the size based on the screen width. I tested it manually on both a Linux desktop and Android tablet. If there are unit tests to write or other required changes before accepting, please let me know.

Tienisto commented:
Thank you.
I think the title "URL:" would fit better than "Zoom".
I played around a bit to avoid the hard coded font size but I couldn't find a better solution yet

harriseldon commented:
Thanks for the quick feedback. Sorry it took me so long for a simple text change. I updated the i18 json file and changed the title to URL per your request.

Tienisto commented:
Thank you
